src/templates/person.js

```diff
@@ -113,11 +113,17 @@ const Person = ({ data }) => {
             </dl>
             <dl>
               <dt>Skills:</dt>
-              {advancedSkills.map((skill, i) => (
-                <dd key={`a${i}`}> {skill};</dd>
+              {advancedSkills.map((skill, i, arr) => (
+                <dd key={`a${i}`}>
+                  {skill}
+                  {arr.length - 1 !== i && <span className="skill-separator"> | </span>}
```

Comment:
I'm being picky but could you please invert the boolean expression ? Much more readable the other way around 
(i !== arr.length - 1)

Reply:
done :)